README.md

```diff
@@ -316,7 +318,39 @@ If you use arbitration graphs in your research, we would be pleased if you cite
   doi = {10.48550/arXiv.2411.10170},
   url={https://arxiv.org/abs/2411.10170}, 
 }
+```
+{% endraw %}
+
+
+<details>
+<summary>Earlier publications</summary>
```

Comment:
The entire section does not render properly for me via Jekyll. Looks fine on the pure markdown rendering and generally I think it's a good idea to use spoilers here.